sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -6,10 +6,19 @@
 
 ### Breaking Changes
 
+- Storage names used for checkpoint store have been normalized to match behavior of other eventhubs clients.
```

Comment:
What "other eventhubs clients"? Is this to become more consistent across other languages?

Users of the C++ SDK, who don't have the context about other language SDKs, might be confused by what this breaking change is doing. Maybe this can be re-worded to mention the change itself, so it is directly applicable to existing beta users.

Reply:
I think it is a reasonable expectation that developers using the C++ SDK are aware of other Azure language SDKs.

I'll update the wording to say "of other Azure SDK eventhubs packages".

The actual change itself is irrelevant because it is an implementation detail.

Reply:
Can you point me to the exact change this CL entry is referring to?

> The actual change itself is irrelevant because it is an implementation detail

To reduce signal-to-noise, if it isn't impacting the end user, we shouldn't mention it as a breaking change in the CL. What action do you expect the reader to take? If none since this is a change in the implementation detail, we can remove it.